corehq/apps/accounting/models.py

```diff
@@ -3821,7 +3821,7 @@ def clean(self):
 class DomainUserHistory(models.Model):
     """
     A record of the number of users in a domain at the record_date.
-    Created by task calculate_users_and_sms_in_all_domains on the first of every month.
+    Created by task calculate_web_users_in_all_billing_accounts on the first of every month.
```

Comment:
is this comment still correct?

Reply:
I used the wrong task name 🤦 
Thanks for catching it. Fixed in https://github.com/dimagi/commcare-hq/pull/32747/commits/bf4655c601521de99b06f5730c3b5acd5ec40ffd